src/main/java/io/vertx/ext/hawkular/VertxHawkularOptions.java

```diff
@@ -81,6 +83,16 @@
    */
   public static final boolean DEFAULT_METRICS_BRIDGE_ENABLED = false;
 
+  /**
+   * The default number of metric names to cache in order to avoid repeated tagging requests = 100.
+   */
+  public static final int DEFAULT_TAGGED_METRICS_CACHE_SIZE = 100;
```

Comment:
+1 this is too conservative, a few thousands should be safe

Besides, if the size is too small, the LRU cache might be simply useless, as entries could get evicted over and over again.